dashboards/sentry-software/sentry-software.json

```diff
@@ -0,0 +1,3021 @@
+{
+    "name": "Hardware Sentry",
+    "description": null,
+    "pages": [
+      {
+        "name": "Main",
+        "description": null,
+        "widgets": [
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![Sentry Software Logo](https://dl.sentrysoftware.com/public/otel-dl-link.png)"
+            }
+          },
+          {
+            "title": "Monitoring Coverage",
+            "layout": {
+              "column": 3,
+              "row": 1,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "FROM(FROM Metric \n  SELECT filter( latest(`hardware_sentry.connector.status`), WHERE state = 'ok' AND host.name IN ({{sentry_host}}) AND site IN ({{sentry_site}}) ) as 'status'\n  FACET site,host.id LIMIT 200) SELECT sum (`status`)*100/(FROM Metric SELECT cardinality(hardware_sentry.host.configured) SINCE 24 hours ago WHERE site IN ({{sentry_site}})) as '% coverage' SINCE 24 hours ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "thresholds": [
+                {
+                  "alertSeverity": "WARNING",
+                  "value": 90
+                },
+                {
+                  "alertSeverity": "CRITICAL",
+                  "value": 50
+                }
```

Comment:
Can we trigger the alert if it exceeds 90% (critical) and 85% (warning) threshold?

Reply:
This is a monitoring coverage %, so the value ideally should be above 90%, and 50-90% means there is an issue. so below 50% would be critical. Also, these are not alerts, this is just for the color coding on the dashboard. if the % is above 90, it shows green, if it's 50 - 90, it shows yellow, if it's below 50, it shows as red.